src/components/CreateButton/index.js

```diff
@@ -8,13 +8,33 @@ const getUniqueID = () => {
     return s4() + s4() + '-' + s4();
 };
 
+const styles = {
+    createButton: {
+        marginTop: '30px',
+        marginBottom: '30px',
+        backgroundColor: '#74aec1',
+        border: 'none', 
+        color: '#242222', 
+        borderRadius: '100px',
+        width: '200px',
+        display: 'inline', 
+        float: 'left',
+        marginLeft: '140px',
+        padding: '12px',
+        fontWeight: '500',
+        textDecorationColor: 'white'
+    }
+};
+
 function CreateButton({ name }) {
     return (
-        <Link onClick={event => (!name) ? event.preventDefault() : null} to={`game?name=${name}&room=${getUniqueID()}`} >
-            <Button variant="primary" size="lg" block>
+        <div className='createButton'>
+            <Link onClick={event => (!name) ? event.preventDefault() : null} to={`game?name=${name}&room=${getUniqueID()}`} >
+                <Button size="lg" style={styles.createButton} size="lg" block>
```

Comment:
read the error in the CI (https://github.com/jacobscarfmerrell/myword/pull/11/checks?check_run_id=777493959)
then check out this line

I think you'll see the issue `:P`